packages/eslint-plugin/docs/rules/naming-convention.md

```diff
@@ -277,6 +280,9 @@ Group Selectors are provided for convenience, and essentially bundle up sets of
 - `variableLike` - matches the same as `function`, `parameter` and `variable`.
   - Allowed `modifiers`: `async`, `unused`.
   - Allowed `types`: none.
+- `accessor` - matches the same as `classicAccessor` and `autoAccessor`.
+  - Allowed `modifiers`: `abstract`, `override`, `private`, `protected`, `public`, `requiresQuotes`, `static`.
+  - Allowed `types`: `array`, `boolean`, `function`, `number`, `string`.
```

Comment:
[Style] The list is in alphabetical order to make it easier to scan through. Let's move this to the top.